### DISCUSSION TITLE
Direct Upgrade to DivestOS 19.1

### DISCUSSION THREAD

**Discussion by TommyTran732** [Divested-Mobile/DivestOS-Build#124] [Q&A] [answered]
Hi,

So I currently have a OnePlus 7T running DivestOS 18.1. I see the new DivestOS 19.1 update in the updater, but there is no option to actually download the update and install it.

Is there any plan for direct upgrade from 18.1 to 19.1 in the future? Is this a bug? Or should I just reset my phone and Flash DivestOS 19.1 manually?

**Comment by SkewedZeppelin** [comment, marked as answer]
Just adb sideload 19.1

No you don't have to wipe.

The button is disabled purposely to ensure users read the website and make a
backup first.
